pkgs/applications/editors/vim/plugins/generated.nix

```
@@ -16479,5 +16479,17 @@ final: prev:
     meta.homepage = "https://github.com/jhradilek/vim-snippets/";
   };
 
+  nvim-bacon = buildVimPlugin {
+    pname = "nvim-bacon";
+    version = "2024-02-12";
```

See https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming about versioning

```suggestion
    version = "0-unstable-2024-02-12";
```

---

vim plugins typically dont make releases but the naming is interesting for `buildNeovimPlugin` since we could see from what version of the lua package it was generated. @GaetanLepage it should be easy to tweak the version in `buildNeovimPlugin`, are you interested in doing this ? might be another PR though

---

Addressed in #289008